packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -32,6 +32,17 @@ enum PlatformViewHitTestBehavior {
   transparent,
 }
 
+/// How an Android view is composed along side with Flutter UI.
+enum AndroidViewCompositeMode {
```

Comment:
What do you think about calling this "SurfaceType" and have the values be texture/platformViewLayer ? as this is what the framework knows about (e.g may end up using a different approach on to composite platform view layers one day)